packages/parser/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "parser",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"build": "rimraf dist && tsc",
+		"start": "node dist/bin/index.js",
+		"dev": "npm run build && env-cmd npm start"
+	},
+	"keywords": [],
+	"author": "",
+	"license": "UNLICENSED",
+	"dependencies": {
+		"@spectacles/brokers": "^0.8.0",
+		"lexure": "^0.4.2",
+		"postgres": "^1.0.2",
+		"reflect-metadata": "^0.1.13",
+		"tslib": "^2.0.0"
```

Comment:
I've never used this before, but it looks like it's intended for use with `importHelpers` which isn't used in this package.

Reply:
`for await ()` emits `__asyncValues` and since we have `noEmitHelpers` we use the tslib helper library.